2017/11/29/systemd-primer-from-the-trenches.html.md

```diff
@@ -0,0 +1,514 @@
+---
+author: Ian Neilsen
+title: "systemd: A primer from the trench's"
+tags: hosting, systemd, systemctl 
+---
+
+# systemctl - lets get back to basics
+
+''help me systemd you are my only hope''. If we start back at the beginning, sometimes going back to day dot
+often brings clarity to what seems like hopeless or frustrating situation for users from SysV world. Caveat, 
+I previously worked at Red Hat for many years before joining the excellent team at End Point and have been using systemd for as long. 
+I quite honestly have forgotten most of the SysV or init days. Although at End Point we work daily on Debian, Ubuntu, Centos and BSD variants.
```

Comment:
It would be good here to note which versions of CentOS, Debian, and Ubuntu first introduced systemd, in case readers don't know which they're running. And since you mention upstart and Ubuntu later, it would be good to explain that Upstart was in RHEL 6 and Ubuntu < 16.04 (I think), so people know when that was around.

Reply:
Can do. I didnt want to pull in too much history.